scripts/optimize_api.py

```
@@ -0,0 +1,53 @@
+import sys
+
+with open('inc/api.php', 'r') as f:
```

Ruff (0.15.1)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)